lib/spreewald/web_steps.rb

```diff
@@ -239,28 +230,20 @@
 
 # Like `Then I should see`, but with single instead of double quotes. In case
 # the expected string contains quotes as well.
-Then /^(?:|I )should see '([^']*)'$/ do |text|
-  patiently do
-    expect(page).to have_content(text)
-  end
-end.overridable
+Then /^(?:|I )should( not)? see '([^']*)'$/ do |negate, text|
+  expectation = negate ? :not_to : :to
 
-Then /^(?:|I )should not see '([^']*)'$/ do |text|
   patiently do
-    expect(page).not_to have_content(text)
+    expect(page).send(expectation, have_content(text))
   end
 end.overridable
 
 # Check that the raw HTML contains a string
-Then /^I should see "([^\"]*)" in the HTML$/ do |text|
-  patiently do
-    expect(page.body).to include(text)
-  end
-end.overridable
+Then /^I should( not)? see "([^\"]*)" in the HTML$/ do |negate, text|
+  expectation = negate ? :not_to : :to
 
-Then /^I should not see "([^\"]*)" in the HTML$/ do |text|
   patiently do
-    expect(page.body).not_to include(text)
+    expect(page.body).send(expectation, include(text))
   end
```

Comment:
It would be really good to have at least some tests for the negated steps. At least some of them are already tested implicitly but I'm not sure if that's good enough?

Reply:
I added a few tests in my last commit.

Reply:
Thank you!